### PR TITLE
feat: added changes and syntax highlighting

### LINKED ISSUE #12: Add syntax highlighting
We should add syntax highlighting.
Here's the docs from react-diff-view:
https://github.com/otakustay/react-diff-view#token-system
And an example provided by them:
https://codesandbox.io/s/react-diff-view-mark-edits-demo-8ndcl?file=/src/index.js

### PR DESCRIPTION
**Pull request by PiotrekPKP**
### Changelog
- added changes highlighting
- **added syntax highlighting** (closes #12)

image

### CODE REVIEW

**Comment by PiotrekPKP** [comment]
Noe: don't upgrade `refractor` lib, as [v4.x is not supported](https://github.com/otakustay/react-diff-view#token-system)

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -3,6 +3,7 @@ import { Inter } from "next/font/google";
 
 import "@/styles/globals.css";
 import "react-diff-view/style/index.css";
+import "../styles/prismjs.css";
```
Comment (andreifilip123) [inline src/pages/_app.tsx]:
```suggestion
import "@/styles/prismjs.css";
```

File: src/pages/diff/[slug].tsx [hunk 2]
```diff
@@ -156,8 +150,25 @@ const DiffPage: NextPage<{
           </div>
         </button>
         {isExpanded && (
-          <Diff viewType={viewType} diffType={type} hunks={hunks}>
-            {(hunks) => hunks.map(renderHunk)}
+          <Diff
```
Comment (andreifilip123) [inline src/pages/diff/[slug].tsx:153]: why did you remove the renderHunk method ?

Comment (PiotrekPKP) [reply src/pages/diff/[slug].tsx:153]: I've moved it into the child part so I could get:
- inherited types
- I could use index for keys (React was yelling at me for duplicate keys in the map)

Comment (andreifilip123) [reply src/pages/diff/[slug].tsx:153]: Alright, sounds good 👌

**Review by andreifilip123 — APPROVED** [review]
Tyvm for your help 🙏 Working on OSS is fun lol! I really like everyone coming along and helping with this 🥰

**Pull request merged**